### DISCUSSION TITLE
Issue with VERY few results for keyword-based search

### DISCUSSION THREAD

**Discussion by chiarasemenzin** [danielnsilva/semanticscholar#50] [Q&A] [answered]
Hello, I'm currently facing an issue with the SS API results. 
When querying the database from the website I get thousands of relevant results for a search such as "l-Acetyl-carnitine", well ranked by relevance.
Nonetheless, from the semanticscholar API I only get the following output:

![Screen Shot 2023-04-21 at 10 22 00](https://user-images.githubusercontent.com/15186930/233585538-196b6230-9444-4266-8910-badc3b9d3a77.png)

Do you have any idea of what might be causing this mismatch?
Thank you in advance for the help!

**Comment by danielnsilva** [comment, marked as answer]
Hello, I think the answer you're looking for is available at this link: https://github.com/allenai/s2-folks/issues/42
